akka-cluster-sharding/src/test/resources/synthetic-loop.conf

```diff
@@ -0,0 +1,68 @@
+#
+# Run with synthetically generated access events with a looping scan through ids.
+#
+# > akka-cluster-sharding/Test/runMain akka.cluster.sharding.passivation.simulator.Simulator synthetic-loop
```

Comment:
would it be interesting to add a synthetic-loop that access the entity with a few messages before moving on to next entity in the loop?